brage-import/src/main/java/no/sikt/nva/brage/migration/merger/findexistingpublication/TitleAndTypePublicationFinder.java

```diff
@@ -125,9 +123,8 @@ private UriWrapper getStandardSearchUri(Publication publication) {
         return UriWrapper.fromHost(apiHost)
                    .addChild(SEARCH)
                    .addChild(RESOURCES)
-                   .addQueryParameter(TITLE, getMainTitle(publication).get())
-                   .addQueryParameter(CONTEXT_TYPE, getPublicationContextType(publication))
-                   .addQueryParameter(AGGREGATION, NONE);
+                   .addQueryParameter(TITLE_SHOULD, getMainTitle(publication).get())
+                   .addQueryParameter(CONTEXT_TYPE, getPublicationContextType(publication));
     }
```

Comment:
We do not mock whole call in test, so no existing tests that fail. Should consider to mock exact api calls in test setup.